Orange/widgets/visualize/tests/test_owmosaic.py

```diff
@@ -194,3 +194,16 @@ def test_does_not_crash(self):
         data = Table("housing.tab")
         widget.set_data(data)
         vizrank.toggle()
+
+    def test_zerodivision_error(self):
+        """
+        ZeroDivisionError should not be thrown when
```

Comment:
Could you make this description more explicit. Instead of "some kind of invalid data" try to describe what the problem in data was.